data_architect_misc/data_transformer/configs/APAC_Taiwan/step2_transform_sos_raw_investment_data.json

```diff
@@ -12,6 +12,7 @@
     "write_output": true,
     "output_folder_path": "./output/APAC_Taiwan/",
     "output_file_name_prefix": "Transformed_Taiwan_20200501_20200531_",
+    "output_encoding" : "utf-8-sig",
```

Comment:
Same as above comment. Leave it as "utf-8" if possible. If not, please ping me on Teams and explain why it's not possible.

Reply:
if I left the utf-8 configured in the config file, the output file showed up with strange character but with the utf-8-sig the characters appears correctly

Reply:
Please leave a comment before "output_encoding" like below:
"__output_encoding_comment__": "In Taiwan input files, we have to use utf-8-sig as output encoding or else it does not work. The character outputs become incorrect if we use utf-8."